docs/known-issues.md

```diff
@@ -9,7 +9,6 @@ Known issue = any major user-facing bug or lack of user-facing feature that:
 | The Unity Editor randomly crashes on MacOS when using ECS preview packages. This is not caused by the SpatialOS GDK.                                                                                                                                                                                                         | 2018/07/25 | 1064084      | Simply retry.                                                         | No     |
 | IL2CPP does not currently work on Mac as it crashes upon starting the game. This is due to a bug in Unity that does not initialize a system correctly, if one of the injected `IComponentData` contains a `long`.                                                                                                            | 2018/08/31 | 1076596      | Use the Mono compiler.                                                | Yes    |
 | `bool` is not a [blittable type](https://docs.microsoft.com/en-us/dotnet/framework/interop/blittable-and-non-blittable-types), so will cause an error if used to store component data.                                                                                                                                       | 2018/09/17 | None         | Use `BlittableBool` instead.                                          | No     |
-| Accessing component data from an entity query will segfault, therefore requests where `ResultType` is `SnapshotResultType` will be dropped before sending.                                                                                                                                                                   | 2018/09/19 | None         | Don't send entity queries where `ResultType` is `SnapshotResultType`. | No     |
```

Comment:
do we remove the known issues or mark them as resolved? the second one issue is resolved and marked as such, but not removed from the table.

would personally prefer to remove

Reply:
+1 to remove

Reply:
Wait we have a fixed column?

Reply:
it is weird. as said offline, let's just remove it